usr/share/rear/build/default/990_verify_rootfs.sh

```diff
@@ -143,25 +159,25 @@ for binary in $( find $ROOTFS_DIR -type f \( -executable -o -name '*.so' -o -nam
     # (in particular there is nothing about LD_LIBRARY_PATH in usr/share/rear/skel/*).
     # First test the binary explicitly without any LD_LIBRARY_PATH setting inside the recovery system.
     # Continue testing the next binary if this one succeeded (i.e. when it has no 'not found' shared object dependency):
-    chroot $ROOTFS_DIR /bin/bash --login -c "unset LD_LIBRARY_PATH && cd $( dirname $binary ) && ldd $binary" </dev/null 2>/dev/null | grep -q 'not found' || continue
+    chroot "$ROOTFS_DIR" /bin/bash --login -c "unset LD_LIBRARY_PATH && cd $( dirname $binary ) && ldd $binary" </dev/null 2>/dev/null | grep -q 'not found' || continue
```

Comment:
Note: I suspect that `dirname $binary` or other parts of this line will break on paths with blanks in them, which we cannot exclude as we scan the entire file system.

I'm not sure if we should preventively fix that or keep it in mind for the future.

Reply:
The current code contains
```
# FIXME: The following code fails if file names contain characters from IFS (e.g. blanks),
# see https://github.com/rear/rear/pull/1514#discussion_r141031975
# and for the general issue see https://github.com/rear/rear/issues/1372
```
so yes, the current code fails in such cases